autoendpoint/src/settings.rs

```diff
@@ -106,15 +106,19 @@ impl Settings {
 
     /// Initialize the fernet encryption instance
     pub fn make_fernet(&self) -> MultiFernet {
-        let fernets = Self::read_list_from_str(&self.crypto_keys, "Invalid AUTOEND_CRYPTO_KEYS")
+        let keys = &self.crypto_keys.replace('"', "").replace(" ", "");
+        let fernets = Self::read_list_from_str(&keys, "Invalid AUTOEND_CRYPTO_KEYS")
```

Comment:
Definitely a simple fix, but just noting here that something like `serde_json::decode::<Vec<String>>` should work.

Reply:
Yeah, I considered that, but the example we had was not properly formatted JSON. For now, I figured I should apply that fix until I can dig into seeing if we allowed similar values in Python.

Reply:
Ah, ok so you want to allow the previous input as well :+1:

Reply:
Heh, well, if this is going to be a drop-in replacement, it needs to follow the same rules we had before.

Not super happy about it, but it's something we can address later if need be.

Reply:
Checked, and the autopush python [sample file](https://github.com/mozilla-services/autopush/blob/master/configs/autopush_endpoint.ini.sample#L28) also used non-JSON values.